packages/indexer-common/src/allocations/query-fees.ts

```diff
@@ -259,6 +301,7 @@ export class AllocationReceiptCollector implements ReceiptCollector {
         // Remove the batch from the processing queue
         // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
         const batch = this.receiptsToCollect.pop()!
+        this.logger.info(` ========${batch} ============`)
```

Comment:
Is this line needed?

Reply:
can be removed, probably was added to get more info